.github/workflows/jira-issue-sync.yml

```diff
@@ -22,20 +25,27 @@ jobs:
           JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
 
+      # Conditionally change the Epic link if the issue contains the bug label
+      - name: Is this a bug
+        run: echo "EPIC_LINK=TF-7331" >> $GITHUB_ENV
```

Comment:
This is meant to be a reusable workflow, with EXTRA_FIELDS being the mechanism for adding team-specific extra customfields. [See here](https://github.com/hashicorp/terraform-provider-tfe/blob/main/.github/workflows/call-jira-issue-sync.yml#L14-L15)